watertap/unit_models/zero_order/tests/test_blending_reservoir_zo.py

```diff
@@ -105,7 +105,7 @@ def test_solution(self, model):
                 value(model.fs.unit.inlet.flow_mass_comp[t, j]), rel=1e-5
             ) == value(model.fs.unit.outlet.flow_mass_comp[t, j])
 
-        assert pytest.approx(8.4491446e-11, rel=1e-5) == value(
+        assert pytest.approx(1.8449145e-10, rel=1e-5) == value(
```

Comment:
For all intents and purposes, this is in our eyes 0, should we not typically test these as value has to be <=1e-8 ? which is linked to tolerances of our constraints?

Reply:
Yes, probably. But there are lots of small value comparisons in the ZO unit model tests, so I choose to preserve this for consistency.